nion/swift/LinePlotCanvasItem.py

```diff
@@ -511,7 +512,7 @@ def calculate_scalar_xdata(xdata_list: typing.Sequence[typing.Optional[DataAndMe
             if self.__display_frame_rate_id:
                 Utility.fps_tick("prepare_"+self.__display_frame_rate_id)
 
-            colors = ('#1E90FF', "#F00", "#0F0", "#00F", "#FF0", "#0FF", "#F0F", "#888", "#800", "#080", "#008", "#CCC", "#880", "#088", "#808", "#964B00")
+            colors = list({*Color.svg_color_map.values()})
```

Comment:
I'd like to keep the existing color list rather than choosing the first colors listed in the svg_color_map. But we should move the list to `DisplayItem.py` so that it is only defined in one place.